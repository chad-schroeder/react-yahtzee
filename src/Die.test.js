import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Die from './Die';

it('renders without crashing', () => {
  shallow(<Die />);
});

it('matches snapshot of unlocked die', () => {
  let wrapper = shallow(<Die locked={false} />);
  let serialized = toJson(wrapper);
  expect(serialized).toMatchSnapshot();
});

it('matches snapshot after locking die', () => {
  let wrapper = shallow(<Die locked={true} />);
  let serialized = toJson(wrapper);
  expect(serialized).toMatchSnapshot();
});
