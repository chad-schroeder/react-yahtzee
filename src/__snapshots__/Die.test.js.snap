// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot after locking die 1`] = `
<button
  className="Die Die-locked"
  onClick={[Function]}
/>
`;

exports[`matches snapshot of unlocked die 1`] = `
<button
  className="Die"
  onClick={[Function]}
/>
`;
